app/actions/local/channel.ts

```diff
@@ -483,3 +484,56 @@ export async function showUnreadChannelsOnly(serverUrl: string, onlyUnreads: boo
         prepareRecordsOnly: false,
     });
 }
+
+export const checkChannelsDisplayName = async (serverUrl: string, preferences?: PreferenceType[]) => {
```

Comment:
this could be 
```suggestion
export const checkChannelsDisplayName = async (serverUrl: string, checkFor: PreferenceType[]|ClientConfig|ClientLicense) => {
```

leaving here as an example